### PR TITLE
fix: update prop name "style" to "variant"

### PR DESCRIPTION
**Pull request by jose-costa-frontify**
Clickup task: [86954cqzc](https://app.clickup.com/t/86954cqzc)

### CODE REVIEW

File: packages/components/src/components/Button/Button.tsx [hunk 1]
```diff
@@ -85,9 +85,9 @@ export const Button = forwardRef<HTMLButtonElement | null, ButtonProps>(
                 type={type}
                 data-test-id={dataTestId}
                 className={cn(
-                    buttonStyles({ size, style, ...props }),
-                    textStyles({ style, ...props }),
-                    iconStyles({ style, ...props }),
+                    buttonStyles({ size, style: variant, ...props }),
+                    textStyles({ style: variant, ...props }),
+                    iconStyles({ style: variant, ...props }),
```
Comment (SamuelAlev) [inline packages/components/src/components/Button/Button.tsx:90]: You can also rename the "style" key inside to "variant"

File: packages/components/src/components/Divider/Divider.tsx [hunk 2]
```diff
@@ -47,6 +53,7 @@ export const DividerComponent = (
             className={cn(
                 dividerStyles({
                     direction,
+                    style: variant,
```
Comment (SamuelAlev) [inline packages/components/src/components/Divider/Divider.tsx:56]: same

File: packages/components/src/components/LoadingBar/LoadingBar.tsx [hunk 3]
```diff
@@ -58,7 +58,7 @@ export const LoadingBar = forwardRef<ElementRef<typeof ProgressRadixPrimitive.Ro
             <ProgressRadixPrimitive.Root
                 ref={ref}
                 data-test-id={dataTestId}
-                className={loadingBarContainerStyles({ rounded, size, style })}
+                className={loadingBarContainerStyles({ rounded, size, style: variant })}
```
Comment (SamuelAlev) [inline packages/components/src/components/LoadingBar/LoadingBar.tsx:61]: same

File: packages/components/src/components/LoadingBar/LoadingBar.tsx [hunk 4]
```diff
@@ -72,7 +72,7 @@ export const LoadingBar = forwardRef<ElementRef<typeof ProgressRadixPrimitive.Ro
                 {...props}
             >
                 <ProgressRadixPrimitive.Indicator
-                    className={loadingBarStyles({ style, indeterminateState: value === null })}
+                    className={loadingBarStyles({ style: variant, indeterminateState: value === null })}
```
Comment (SamuelAlev) [inline packages/components/src/components/LoadingBar/LoadingBar.tsx:75]: same

**Review by SamuelAlev — CHANGES_REQUESTED** [review]

File: packages/components/src/components/LoadingBar/LoadingBar.tsx (hunk 4 above)
Comment (jose-costa-frontify) [reply packages/components/src/components/LoadingBar/LoadingBar.tsx:75]: will do

File: packages/components/src/components/Divider/Divider.tsx (hunk 2 above)
Comment (SamuelAlev) [inline packages/components/src/components/Divider/Divider.tsx:56]:
```suggestion
                   variant,
```

Comment (jose-costa-frontify) [reply packages/components/src/components/Divider/Divider.tsx]: thank you, done!

File: packages/components/src/components/Divider/Divider.tsx [hunk 5]
```diff
@@ -17,7 +17,7 @@ export type DividerProps = {
      * The style of the divider
      * @default "solid"
      */
-    style?: DividerStyle;
+    variant?: DividerStyle;
```
Comment (SamuelAlev) [inline packages/components/src/components/Divider/Divider.tsx:20]: Sorry I just saw it now with CTRL F 😬 
Could you update the type name to `DividerVariant` (same in other files) and also the JSDoc (same in other files)

**Review by SamuelAlev — APPROVED** [review]
Left a comment, after it is fixed, feel free to merge

**Pull request merged**